nodejs-cpp/overview/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Overview
+linktitle: Overview
+type: docs
+weight: 20
+url: /nodejs-cpp/overview/
+description: Overview of key-features and supported formats of Aspose.PDF for Node.js via C++, installation and licensing manual.
+lastmod: "2022-12-13"
+sitemap:
+    changefreq: "weekly"
+    priority: 0.7
+---
+
+_This page is an overview of Aspose.PDF for Node.js via C++ features._
+
+**Aspose.PDF for Node.js via C++** allows developers manipulate them PDF files directly in the Web. This Node.js Solution is built on the use of WebAssembly technology and is based on Aspose.PDF for .NET. You can easily use aspose-pdf for Node.js via C++ from and then follow the installation instructions.
+
+**About CJS and MJS file extensions**
+
+Node.js supports two modular systems for organizing and sharing code. There are:
+
+- CommonJS Modules
+- ECMAScript modules
+
+The CommonJS modules were the classic dependency processing method in Node.js, while the ECMAScript modules are the newer add-on and follow the standard format used in modern browsers. 'cjs', 'mjs' - are extensions used to specify the module formats (CommonJS and ECMAScript Modules) used in JavaScript files. 
+
+In the code snippets, you will notice a difference in work with the AsposePDFforNode.сjs module. The primary distinction between 'require' and 'import' lies in their use cases. '***Require** is limited to importing entire modules, whereas **Import** has the flexibility to import both entire modules and specific exports from those modules.
+
+**Two different approaches**
+
+JavaScript offers two methods, 'then' and 'await', for handling asynchronous operations.
```

Comment:
Remove this or rewrite. It's technically wrong. Please [read this](https://javascript.info/async-await)